packages/experimental/RadioGroup/src/Radio/useRadio.win32.ts

```diff
@@ -51,10 +52,59 @@ export const useRadio = (props: RadioProps): RadioInfo => {
     }
   }, []);
 
+  const isCircularNavigation = true;
+
+  const onInvoke = React.useCallback(
+    (e: KeyPressEvent) => {
+      if (
+        e.nativeEvent.key == 'ArrowDown' ||
+        e.nativeEvent.key == 'ArrowRight' ||
+        e.nativeEvent.key == 'ArrowUp' ||
+        e.nativeEvent.key == 'ArrowLeft'
+      ) {
+        const length = radioGroupContext.enabledValues.length;
+        const currRadioIndex = radioGroupContext.enabledValues.indexOf(radioGroupContext.value);
+        let newCurrRadioIndex;
+        if (e.nativeEvent.key === 'ArrowDown' || e.nativeEvent.key == 'ArrowRight') {
```

Comment:
Does this respect RTL?

Reply:
Based on my understanding, I think yes because arrow key behavior would remain the same regardless of RTL since Radio ordering would remain the same.

Reply:
If the script (and label and all the views..) was reversed, you wouldn't want arrowLeft to go to the next radio button?

Reply:
+1, I think we should double check RTL behavior.

Reply:
Thanks for the callout! Was able to check for RTL and verify changes in FURN tester app via Lime. It should work correctly now.